packages/core/src/compiler.ts

```diff
@@ -513,9 +513,7 @@ export class Compiler<Identity extends Saved | Unsaved = Saved> {
   }
 
   private getSchemaModuleRef(parentCard: CompiledCard | undefined, schemaPath: string | undefined): ModuleRef {
-    if (isBaseCard(this.cardSource)) {
-      return { global: `${BASE_CARD_URL}/schema` };
```

Comment:
I'm not sure how this worked before... maybe an artifact now?

Reply:
if we don't need this, then definitely get rid of it, and also we can remove the base-card logic in the file-cache's module resolver too: https://github.com/cardstack/cardstack/blob/main/packages/hub/services/file-cache.ts#L142-L149